Makefile

```diff
@@ -25,11 +28,18 @@ clean: clean-vectors ## 🧹 Remove build artifacts
 
 STAMP_FILE := .docker_build_stamp
 $(STAMP_FILE): $(shell find crates cmd -type f -name '*.rs') Cargo.toml Dockerfile
-	docker build -t ethrex .
+	docker build -t ethrex --build-arg "BUILD_FLAGS=${--features levm}" .
+	touch $(STAMP_FILE)
+
+STAMP_FILE_REVM := .docker_build_stamp_revm
```

Comment:
Can't we wait for https://github.com/lambdaclass/ethrex/pull/1706 so that we don't have to create this extra code? After 1706 this PR should be trivial

Reply:
I think we can. I'll reduce #1706 scope to just replacing the use of feature flags with the `ethrex` CLI flag so it can be delivered sooner.